stdint.h is a subset of headers of inttypes.h include either or not both.
--- firefox-68.6.0/media/ffvpx/libavutil/common.h.orig  2020-03-20 13:07:29.689029580 +0000
+++ firefox-68.6.0/media/ffvpx/libavutil/common.h       2020-03-20 13:07:37.410068233 +0000
@@ -34,7 +34,6 @@
-#include <inttypes.h>
 #include <limits.h>
 #include <math.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
